sdk/formrecognizer/azure-ai-formrecognizer/tests/test_to_dict.py

```diff
@@ -0,0 +1,1069 @@
+# coding=utf-8
```

Comment:
just a general thought, you might take inspiration from test_repr.py which uses fixtures to re-use a lot of the instantiated objects so we don't have to recreate them for each test. Not saying to update what you have here, but just pointing it out in case it's helpful for later. :)

Reply:
Yeah wasnt aware of how to use that 😅 I'll definitely take a look at that later. At the moment, I simply reused these tests for the from_dict methods as well, since I only had to change one line in each of the tests.